tools/ci_build/github/azure-pipelines/linux-cpu-minimal-build-ci-pipeline.yml

```diff
@@ -22,7 +22,7 @@ jobs:
 
   variables:
     test_data_directory: $(Build.SourcesDirectory)/.test_data
-
+    MYSQL_SERVER_IP: 172.19.0.7
   steps:
```

Comment:
Is there documentation somewhere we could link to regarding this machine? If late one night you decided to immediately steal a super-yacht, cruise the south pacific and never return to work wouldn't anyone else know where this machine is, how to access it , how it was setup, etc. etc.? 

Or more probable, if you were on vacation and the machine stopped responding causing CIs to fail what are our odds of finding it and being able to fix it?

Reply:
Nice plan. I wonder if the water in the south pacific is as cold as the one around SF bay area.  If that's the case, I need to buy a wetsuit first. 

I'll send my detailed plan via email to the team.